lib/v8debugapi.js

```diff
@@ -384,10 +384,27 @@ module.exports.create = function(logger_, config_, fileStats_) {
   // }
 
   function findScripts(scriptPath) {
+    // Use repository relative mapping if present
+    if (config.appPathRelativeToRepository) {
+      var candidate = scriptPath.replace(config.appPathRelativeToRepository,
+        config.workingDirectory);
```

Comment:
Did you conscious choose to replace with `config.workingDirectory`? The alternative would be to replace with an empty string. The former assumes that application root == working directory. Thoughts?